src/Models/HydrostaticFreeSurfaceModels/update_hydrostatic_free_surface_model_state.jl

```diff
@@ -21,7 +21,7 @@ hydrostatic pressure) to the current model state.
 """
 update_state!(model::HydrostaticFreeSurfaceModel) = update_state!(model, model.grid)
```

Comment:
Don't we need to pass callbacks through this function?

Reply:
it's like this for all other models... so I was confused.

Reply:
Yes, it's changed! :) see [8a678d8](https://github.com/CliMA/Oceananigans.jl/pull/2786/commits/8a678d8c2c2c53844f773f529a5398aee12f032a)